projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -48,53 +49,82 @@ export default class DeployJobDialog extends Component<IJobFullProps> {
       </>
     );
   }
+
+  handleCheckboxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Yes, one of the with the window element was a previous implementation (i didn't realise it was left there since it does not do anything). i have deleted it